frontend/src/modules/otherlocales/selectors.js

```diff
@@ -0,0 +1,48 @@
+/* @flow */
+
+import { createSelector } from 'reselect';
+
+import type { OtherLocaleTranslation, OtherLocaleTranslations } from 'core/api';
+
+import { NAME } from '.';
+
+const otherLocalesSelector = (state): string => state[NAME].translations;
+
+/**
+ * Return the Entity that follows the current one in the list.
+ */
+
+function _flattenOtherLocales(
```

Comment:
With the current backend response and redux state shape, I had to do this in order to have all other locale translations in a single list. I feel like the backend could already provide this list as-is, and the fact that some locales are preferred could be indicated via an optional `"is_preferred": true` flag perhaps. So I wonder what the motivation of the current backend response is. Any ideas?

Reply:
Cool, I could handle that in a follow-up as well.

Reply:
Great! Filed https://bugzilla.mozilla.org/show_bug.cgi?id=1686690.